src/http/request.cr

```diff
@@ -98,6 +98,19 @@ class HTTP::Request
     value
   end
 
+  # Return request host from headers
+  def host
+    host = @headers["Host"]?
+    return unless host
+    index = host.index(":")
+    index ? host[0...index] : host
```

Comment:
@jhass I guess you're [right](https://en.wikipedia.org/wiki/IPv6_address#Presentation). But still, it seems hacky. Why not use `URI`? This is how security issues are introduced, by doing things that are outside the responsibilities of the affected class. Parsing an URI is, IMHO, outside of the scope of `HTTP::Request` and `URI` (well, `URI::Parser`) most likely does a better job at it.